pkg/anchor/docker.go

```diff
@@ -82,12 +94,55 @@ func processRunCommand(ctx context.Context, node *Node, architecture string, ima
 	return nil
 }
 
+func parseComment(entry Entry) ([]string, bool) {
+	ignoredPackages := []string{}
+	if entry.Type != EntryComment {
+		return ignoredPackages, false
+	}
+
+	command := strings.TrimLeft(entry.Value, "# ")
+	commands := strings.SplitN(command, " ", 2)
+	if len(commands) < 2 {
+		return ignoredPackages, false
+	}
+	if strings.TrimSpace(commands[0]) != "anchor" {
+		return ignoredPackages, false
+	}
+
+	next := strings.SplitN(commands[1], "=", 2)
+	if len(next) < 2 {
+		if strings.TrimSpace(next[0]) == "ignore" {
+			return ignoredPackages, true
+		}
+		return ignoredPackages, false
+	}
+
+	if strings.TrimSpace(next[0]) != "ignore" {
+		return ignoredPackages, false
+	}
+
+	packages := strings.Split(next[1], ",")
+	for _, pkg := range packages {
+		ignoredPackages = append(ignoredPackages, strings.TrimSpace(pkg))
+	}
+
+	return ignoredPackages, false
+}
+
 func appendPackageVersions(node *Node, packageMap map[string]string, architecture string) {
 	aptGet := false
 	install := false
 	dpkgSet := false
+	ignoredPackages := []string{}
 	for i := range node.Entries {
```

Comment:
Might be out of scope for this PR (can do in a new one), but we should replace the `i` and `j` variables with more meaningful names

Reply:
IMO since `i` and `j` are indexes, it's perfectly fine to keep them like that. I can modify it if you wish though.